cpp/dolfinx/fem/CoordinateElement.cpp

```diff
@@ -264,101 +208,105 @@ CoordinateElement::tabulate(int n, const array2d<double>& X) const
   auto _X = xt::adapt(X.data(), X.shape);
   return _element->tabulate(n, _X);
 }
-//-----------------------------------------------------------------------------
-
-void CoordinateElement::compute_jacobian_data(
-    const xt::xtensor<double, 4>& tabulated_data, const array2d<double>& X,
-    const array2d<double>& cell_geometry, std::vector<double>& J,
-    xtl::span<double> detJ, std::vector<double>& K) const
+//--------------------------------------------------------------------------------
+void CoordinateElement::compute_jacobian(
+    const xt::xtensor<double, 4>& tabulated_data,
```

Comment:
Use a more meaningful argument name.

Reply:
Changed to dphi. We only need the first derivative of the shape functions.